### PR TITLE
UUIN-308 Add closedByDefault prop to TagsAccordion

### PR DESCRIPTION
**Pull request by AndreiShumski2**
Add `closedByDefault` prop to `<TagsAccordion />` for managing open/close accordion status by default

### CODE REVIEW

**Review by uladislausamets — APPROVED** [review]

File: lib/Tags/TagsAccordion.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const TagsAccordion = React.memo(({ link, getEntity, getEntityTags, entityTagsPa
             <FormattedNumber value={entityTags.length} />
           </Badge>
         )}
+        id={id}
```
Comment (aliaksei-chumakou) [inline lib/Tags/TagsAccordion.js:35]: Have you tried to use `closedByDefault` prop? IMHO, it's much more clear solution

Comment (AndreiShumski2) [reply lib/Tags/TagsAccordion.js:35]: Maybe it's better to add `{...rest}` props to the accordion to have the possibility to pass other props?

Comment (aliaksei-chumakou) [reply lib/Tags/TagsAccordion.js:35]: I think it's a common requirement - to show accordion opened initially if some tags are selected. Maybe you can check `entityTags` for length

File: lib/Tags/TagsAccordion.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ const TagsAccordion = React.memo(({ link, getEntity, getEntityTags, entityTagsPa
             <FormattedNumber value={entityTags.length} />
           </Badge>
         )}
+        closedByDefault={!entityTags.length}
```
Comment (aliaksei-chumakou) [inline lib/Tags/TagsAccordion.js:35]: to be on the safe side I would put `!entityTags.length`
```suggestion
        closedByDefault={!entityTags?.length}
```

**Review by aliaksei-chumakou — APPROVED** [review]

**Pull request merged**